apigee_devportal_kickstart.install

```diff
@@ -704,3 +705,36 @@ function apigee_devportal_kickstart_update_9002() {
   // Clear all caches.
   drupal_flush_all_caches();
 }
+
+/**
+ * Updating install and optional configs for callback url.
+ */
+function apigee_devportal_kickstart_update_9003() {
```

Comment:
As there is already an update script in [apigee-edge PR](https://github.com/apigee/apigee-edge-drupal/pull/913), do we have to add it again?

Reply:
As dicussed in call, I have removed the update

Reply:
done in update